docs/concepts/json_schema.md

```diff
@@ -660,32 +660,24 @@ print(json.dumps(ta.json_schema(), indent=2))
     [`pydantic.json_schema.WithJsonSchema`][pydantic.json_schema.WithJsonSchema]<br>
 
 !!! tip
-    Using [`WithJsonSchema`][pydantic.json_schema.WithJsonSchema]] is preferred over
+    Using [`WithJsonSchema`][pydantic.json_schema.WithJsonSchema] is preferred over
     [implementing `__get_pydantic_json_schema__`](#implementing_get_pydantic_json_schema) for custom types,
     as it's more simple and less error-prone.
 
 The [`WithJsonSchema`][pydantic.json_schema.WithJsonSchema] annotation can be used to override the generated (base)
 JSON schema for a given type without the need to implement `__get_pydantic_core_schema__`
-or `__get_pydantic_json_schema__` on the type itself.
-
-This provides a way to set a JSON schema for types that would otherwise raise errors when producing a JSON schema,
-such as `Callable`, or types that have an [`is-instance`][pydantic_core.core_schema.is_instance_schema] core schema.
-
-For example, the use of a [`PlainValidator`][pydantic.functional_validators.PlainValidator] in the following example
```

Comment:
I couldn't find a valid example where `WithJsonSchema` can be useful with validators/serializers, as we know provide `json_schema_input_type`. Let me know if you can find one